### PR TITLE
Added a barricade plushie 

### PR DESCRIPTION
**Pull request by carlarctg**
# About the pull request

Added squeak plush noises used both on the new plush and on the farwa dolls.

Added the barricade plushie to alpha-bravo dorms, sneakily hidden as a little easter egg.

Added a helpful inhands template to the files.

# Explain why it's good for the game

> Added squeak plush noises used both on the new plush and on the farwa dolls.

Sounds are a great and surprisingly easy way to add immersion to CM.

> Added the barricade plushie to alpha-bravo dorms, sneakily hidden as a little easter egg.

This is a little fun toy for marines to mess around with, like the super top hat. Sprites by Sammy, a very, very long time ago.

Added a helpful inhands template to the files.

Old MR:

https://gitlab.com/cmdevs/colonial-warfare/-/merge_requests/1660

Nanu, Triio, and Monkeys approved this back then, so it certainly fits in the CM world. Hard dels PR coming up next.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl: Carlarc, Nauticall
soundadd: Added squeak plush noises used both on the new plush and on the farwa dolls.
imageadd: Added a new plushie, hidden somewhere.
code: Added a helpful inhands template to the files.
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
> You currently have a negative Fix/Feature pull request delta of -23. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

what the FUCK why is it so low

gonna make some refactor PRs now - merge my gun refactor please, i need to continue work based on that

**Comment by NewyearnewmeUwu** [comment]
what does the squeaky sound like

File: code/game/objects/items/toys/toys.dm [hunk 1]
```diff
@@ -472,13 +472,31 @@
...
 	w_class = SIZE_TINY
 	icon_state = "predplush"
 
+/obj/item/toy/plushie_cade
+	name = "plushie barricade"
+	desc = "Great for squeezing whenever you're scared. Or lightly hurt. Or in any other situation."
+	icon_state = "plushie_cade"
+	item_state = "plushie_cade"
+	w_class = SIZE_SMALL
+	var/last_hug_time
```
Comment (Zonespace27) [inline code/game/objects/items/toys/toys.dm:489]: please check if we have cooldown macros (`DECLARE_COOLDOWN` [maybe it's backwards, I don't remember] and etc.), and change this to that if so

also, codedoc comment

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/toys/toys.dm (hunk 1 above)
Comment (harryob) [reply code/game/objects/items/toys/toys.dm:489]: we do btw

**Comment by Ultimate-Fluff** [comment]
image
He's such a smiley boi and I'm all for it.

**Review by Zonespace27 — APPROVED** [review]

**Comment by harryob** [comment]
CleanShot 2023-02-22 at 20 49 09

**Pull request merged**